libafl_libfuzzer/src/lib.rs

```diff
@@ -1,5 +1,68 @@
 //! `libafl_libfuzzer` offers a "permanent" replacement for the now-deprecated libfuzzer
 //!
+//! ## Usage
```

Comment:
Want to copy&paste this into the book as well? might make sense (or include it from there)

Reply:
Or maybe it's too technical

Reply:
Not too technical. Makes sense to include it since many folks might be coming from libfuzzer.

Reply:
What section should it be put under? Its own? It can be used with both C/C++ and Rust targets but needs different setups.